Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,41 @@
+---
+title: _Descriptive title_
+type: Decision Record
+status: Draft
+track: Global  # | IaaS | Ops | KaaS | IAM
+author: Author <Mail or Github handle>
+date: DD-MM-YYYY
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
+See https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md
+--->
+
+## Introduction
```

Comment:
Do we actually need an introduction section for a DR? I can see it making sense for a standard but if I imagine the situation of needing to write a DR in my head, I would want the process to be short and concise. I can't imagine anything I'd want to write here when documenting a decision since I'd put any surrounding information into the "Context" section anyways.

Reply:
https://github.com/SovereignCloudStack/standards/pull/441/commits/1795a118b1bbc30bab68d2b64c36f30d267be099

Reply:
As discussed in the SIG Standardization/Certification today, we will reintroduce this section as "Abstract" aiming to briefly introduce what the decision (record) is about in one or two sentences on a high level. "Context" would then be used to more deeply describe technical details.

I'm fine with this separation. Resolving this thread.